Components/Contact/Contact.tsx

```diff
@@ -0,0 +1,136 @@
+import { Promo } from "../Promo/Promo";
+import CustomButton from "../CustomButton";
+import { Flex, Text, useBreakpointValue } from "@chakra-ui/react";
+import Call from "@/public/icons/call.svg";
+export const Contact = () => {
+  const contactText = useBreakpointValue({
+    base: (
+      <Text
+        textAlign="center"
+        maxW="640px"
+        lineHeight="20px"
+        fontWeight={500}
+        color={"rgba(113, 52, 136, 1)"}
+        fontSize="14px"
+      >
+        تميز الأكاديمية العربية للبرمجة بتطوير أدوات
```

Comment:
avoid using <br /> and this is not an efficient way to handle text use CSS instead